### ISSUE TITLE
[Bug]: credential propagation was unsuccessful

### ISSUE THREAD

**Issue by Salvia2023** [camenduru/stable-diffusion-webui-colab#355]
### What happened?

Hello, this is not a bug. However, I would like to know if this is a problem that you can solve.

Due to the usage limitations of the free version of Colab and even the paid version, I have created three additional accounts, for a total of four accounts, to research and produce more AI images as efficiently as possible. However, due to the usage limits of Colab, I have had to think of other ways to work around these limitations.

One method is to use my other accounts that have 2TB of cloud storage to compute images. The problem is that I only have 4 accounts that have 2TB of cloud storage, and the others do not. Therefore, I thought that when the code **"drive.mount('/content/drive')"** requests access to Google Drive, I could choose to access the accounts that have 2TB of cloud storage. However, this idea failed, and the system displayed **"MessageError: Error: credential propagation was unsuccessful."**

In short, this method is using Colab account A to access Google Drive account B. However, it is possible that due to limitations set by Google or the code, this cross-account credential cannot work.

Therefore, I would like to know if you can help me solve this credential issue. Thank you very much.

### Colab cell output

```shell
MessageError: Error: credential propagation was unsuccessful.
```

### Which colab and model(s) were you using when the error occurred?

https://github.com/camenduru/stable-diffusion-webui-colab/tree/drive

### Which Public WebUI Colab URL were you using when the error occurred?

gradio.live

**Comment by Salvia2023** [comment]
> Hi @Salvia2023 please share `2TB of cloud storage` folder to `other drives ` maybe works

OK, thanks for your suggestion, I'll try it.
